# represent the object belief state
uint32 obj_id
geometry_msgs/Pose pose
VoxelTSDF tsdf
